data/campaigns/Heir_To_The_Throne/scenarios/18_A_Choice_Must_Be_Made.cfg

```
@@ -213,7 +213,7 @@
                 [/message]
                 [message]
                     speaker=Kalenz
-                    message= _ "With all due respect, <i>Princess</i>, it is once again we who will spare you, Sceptre or no Sceptre."
+                    message= _ "With all due respect, Your <i>Highness</i>, it is once again we who will spare you, Sceptre or no Sceptre."
```

We'll need translation hints, for the languages where "Your Highness" needs to take account of the subject's gender.

---

Added

---

I left this one as "Princess" intentionally. Kalenz is well aware of how to address other nobles formally and chooses to call Li'sar 'Princess' in this instance.

---

Why does he address her that way? What's he saying between the lines?

---

That she's acting bratty and stupid. The following line tells us that he definitely got on Li'sar's nerves a bit, since she threatens to fry him. So I was thinking she took his deliberate mistitling as a direct insult, which it would be.

---

I don't think it would be an _insult_ so much as a faux pas. It's still respectful, it's still acknowledging her title, it's just not the correct way to do so. To be an insult wouldn't it have to be "brash princess" or some such?

Point taken about Li'sar's threat to fry him, but it's possible that she was responding just to the "We'll spare you" part, not to the form of address.

In any case, once we decide what's going on here I'll add a po comment documenting the intention.

---

> To be an insult wouldn't it have to be "brash princess" or some such?

And note that he addresses her as "whelp" later:

https://github.com/wesnoth/wesnoth/blob/ec3b2fb381d17a9297ae5f9a2c1ff9fa16649b75/data/campaigns/Heir_To_The_Throne/scenarios/18_A_Choice_Must_Be_Made.cfg#L332-L335

---

Well I tend to be a bit subtle (read, maybe too subtle) when I write things, so up to you if you want to change it.

---

There's a collequal usage of "princess" in British English, meaning referring to a child in the context that their parents think of them. I read Kalenz's statement as if he was a parent telling off one of his own daughters - it would be an insult even if she wasn't royalty.

---

@stevecotton I know that usage, but I thought that one wouldn't be capitalized. It does fit quite well, though, especially given that Li'sar's troops are no longer with her at this point. (Come to think of it, there should be a dialog between Li'sar and one of her generals at the end of S16, where the general deserts her and reports back to Asheviere, to set up the line in S22 where she's called a traitor.) So... this?

```suggestion
                    # po: "Princess" is not a royal title here, but a colloquial usage, meaning referring to a child in the context that their parents think of them. Kalenz is deliberately belittling Li'sar.
                    message= _ "With all due respect, <i>princess</i>, it is once again we who will spare you, Sceptre or no Sceptre."
```

---

Thanks, committed that and created https://github.com/wesnoth/wesnoth/issues/4401 for S16.